synfig-studio/src/gui/app.cpp

```diff
@@ -1396,7 +1400,6 @@ App::App(const synfig::String& basepath, int *argc, char ***argv):
 		synfig::error("UNABLE TO CREATE \"%s\"",synfigapp::Main::get_user_app_directory().c_str());
 	}
 
-
 	ipc=new IPC();
```

Comment:
For your information, the IPC class can be deleted once Gtk:: Application is working. :D